consensus-peer/connection/grpc-logger.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright IBM Corp. 2016 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		 http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package connection
+
+import (
+	"github.com/op/go-logging"
+	"google.golang.org/grpc/grpclog"
+)
+
+type lowVerbosityGRPCLog struct {
+	*logging.Logger
+}
+
+func (l *lowVerbosityGRPCLog) Fatalln(args ...interface{}) {
+	l.Fatal(args...)
+}
+
+func (l *lowVerbosityGRPCLog) Print(args ...interface{}) {
+	l.Debug(args...)
+}
+
+func (l *lowVerbosityGRPCLog) Printf(format string, args ...interface{}) {
+	l.Debugf(format, args...)
+}
+
+func (l *lowVerbosityGRPCLog) Println(args ...interface{}) {
+	l.Debug(args...)
+}
+
+func init() {
+	l := &lowVerbosityGRPCLog{logging.MustGetLogger("grpc")}
+	logging.SetLevel(logging.INFO, "grpc")
```

Comment:
Yes, grpc does add a lot of logging information about reconnects

On 08/03/2016 06:18 PM, Jason Yellick wrote:
>
> In consensus-peer/connection/grpc-logger.go
> <https://github.com/corecode/fabric/pull/2#discussion_r73357283>:
>
> > +
> > +func (l *lowVerbosityGRPCLog) Print(args ...interface{}) {
> > +	l.Debug(args...)
> > +}
> > +
> > +func (l *lowVerbosityGRPCLog) Printf(format string, args ...interface{}) {
> > +	l.Debugf(format, args...)
> > +}
> > +
> > +func (l *lowVerbosityGRPCLog) Println(args ...interface{}) {
> > +	l.Debug(args...)
> > +}
> > +
> > +func init() {
> > +	l := &lowVerbosityGRPCLog{logging.MustGetLogger("grpc")}
> > +	logging.SetLevel(logging.INFO, "grpc")
>
> This still seems like a code change? I guess what I'm driving at, is
> why isn't this logging configured in the same way that the rest of our
> logging is? Are you concerned that someone who sets logging
> generically to DEBUG is going to get spammed with too many messages?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/corecode/fabric/pull/2/files/35904278bc43f5d81564c7d58dee9c93f50af3a9#r73357283>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAK3O1PA0O6muAQT7hKzYZXgocMG-G9aks5qcL9vgaJpZM4Jaq4n>.
>